app/controllers/transactions/transaction-list.controller.it.test.js

```diff
@@ -51,9 +51,15 @@ describe('The /transactions endpoint', () => {
       // No mocking defined on purpose to mock a network failure,
       // This integration test will cover server errors outside the 500 and 504 defined in the Cypress test
       await transactionListController(req, res, next)
-      const expectedError = sinon.match.instanceOf(Error)
+      // Temorary replacement for test failing with axios
+      sinon.match.instanceOf(Error)
```

Comment:
Just need to figure this test out.

Reply:
- 'try - catch' clause added in `connector.client.js` in order to throw an error if unable to retrieve transaction or card types.
- associated test reverted to original assertion so, in effect, no amendment needed.